lia-ui/src/main/java/com/lithium/community/android/ui/components/utils/LiUIUtils.java

```diff
@@ -215,4 +215,7 @@ public static void updateMarkReadPopmenuTitle(PopupMenu popup, LiMessage item) {
         }
     }
 
+    public static void showDeleteMessageStatus(Activity activity, boolean successful){
```

Comment:
Unnecessary method.. can be excluded from the PR

Reply:
yes, deleted